spatial-tracking-explainer.md

```diff
@@ -147,20 +147,19 @@ function onSessionStarted(session) {
 }
 ```
 
-#### Position-disabled reference spaces
+### Viewer reference spaces
+A _viewer_ reference space tracks the origin of the viewer. This type of reference space is primarily used for creating inline experiences with no tracking information. Instead, developers may use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms). An example usage of an _viewer_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
```

Comment:
>A _viewer_ reference space tracks the origin of the viewer.

This phrasing sounds a bit strange now... I know this is the explainer, but maybe something about the origin of the viewer reference space always being at the viewer's position and orientation?

Reply:
Especially when the next sentence immediately says "no tracking information".